dato/js/recview.js

```diff
@@ -43,6 +36,48 @@ $(document).ready( function(event){
     // function to construct element tree for recommender view
     // if resp.data.response array is empty it skips creating recommender section
     function renderView(err, resp) {
+
+      // cb event for mouse over event
+      // tracks when user first hovers over recommender section
+      // sends feedback with data
+      function handleView(event) {
+
+        // once we know the schema for what to send back on mouseOver the recommendations
+        // uncomment code and set data
+        var data = {
+          viewed: 1
+        };
+        client.feedback(resp.data.uuid, { data : data }, function(err, resp) {
+          if(err) {
+            console.log(err);
+          }
+        });
+      }
+
+      // onClick event for links in recommender list
+      // parameters: uuid, and itemId
+      // calls feedback funciton in PS client
+      function onClick(uuid, itemId) {
+        // Calls client feedback function and sends uuid and feedbackData
+        var data = {
+          itemId: itemId,
+          click: 1
+        };
+        client.feedback(uuid, { data : data }, function(err, resp) {
+          if(err) {
+            console.log(err);
+          }
+          window.location.replace(itemId);
```

Comment:
Should use `assign` rather than `replace`? http://stackoverflow.com/questions/4505798/difference-between-window-location-assign-and-window-location-replace